zeroknowledgeproof/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: 'com.android.library'
```

Comment:
We can remove this and apply the normal kotlin plugin here right? or does it have android dependencies?

Reply:
No actual android deps afaik, this was generated by AndroidStudio